frontend/server/src/CreateUserParams.php

```diff
@@ -68,8 +80,24 @@ public function __construct($params = []) {
 
         $this->name = $params['name'] ?? null;
 
-        \OmegaUp\Validators::validateEmail($params['email'] ?? null, 'email');
-        $this->email = $params['email'] ?? '';
+        $this->email = null;
+        if (!isset($params['parent_email'])) {
+            if (!isset($params['email'])) {
+                    throw new \OmegaUp\Exceptions\InvalidParameterException(
+                        'parameterEmpty',
+                        'email'
+                    );
+            }
```

Comment:
```suggestion
        if (isset($params['email'])) {
```

we can use positive logic!